### ISSUE TITLE
Saltheril's Haven (Blood Elf quest)

### ISSUE THREAD

**Issue by Solpowered** [dalaranwow/dalaran-wow#4786]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The quest Saltheril's Haven doesn't appear/exist 
**Expected behaviour**: Tell us what should happen instead
The quest should be offered/ given by Magistrix Landra Dawnstrider
**Steps to reproduce the problem**:

1. Be at least level 8 then go to Fairbreeze Village.
2. Talk to Magistrix Landra Dawnstrider.
3. She doesn't offer the quest. 

**Include proofs for this behaviour**
http://www.wowhead.com/quest=9395/saltherils-haven
http://www.wowhead.com/npc=16210/magistrix-landra-dawnstrider
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://www.wowhead.com/quest=9395/saltherils-haven
http://www.wowhead.com/npc=16210/magistrix-landra-dawnstrider
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
You need to do this quest: https://wowgaming.altervista.org/aowow/?quest=9255 (and the other quests mentioned in the questchain) frist in order to accept the quest: Saltheril's Haven.

This works fine on PTR.
